src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css

```diff
@@ -40,4 +40,5 @@
   display: flex;
   column-gap: 5px;
   padding: 4px 10px;
+  --pill-button-font-size: 11px;
```

Comment:
Here and below. If you check stylelint warnings, doesn't it suggest putting variable declarations at the top within a block of rules?